pageserver/src/tenant/config.rs

```diff
@@ -42,6 +42,7 @@ pub mod defaults {
     pub const DEFAULT_WALRECEIVER_LAGGING_WAL_TIMEOUT: &str = "3 seconds";
     pub const DEFAULT_MAX_WALRECEIVER_LSN_WAL_LAG: u64 = 10 * 1024 * 1024;
     pub const DEFAULT_EVICTIONS_LOW_RESIDENCE_DURATION_METRIC_THRESHOLD: &str = "24 hour";
```

Comment:
let's reduce this value to have at maximum 2Gb overhead

Reply:
Ah does it not need to be 0?

Reply:
Or I don't at least understand how will this then work as a feature flag.

Reply:
This parameter allows not only to disable forcing of image layers generation for GC needs at all, but reduce speed at which them are generated, given chance to GC to delete old layers and so avoid explosion of storage size. 

So we have `gc_period`, `compaction_period`, `pitr_interval` and  `forced_image_creation_limit`. At each compaction iteration we can generate up to   `forced_image_creation_limit` * `compaction_target_size` new data. It should pass at least  `pitr_interval` seconds before GC will be able to delete something.
So  `pitr_interval` / `compaction_period` * `forced_image_creation_limit` * `compaction_target_size` - is upper bound for storage growth caused by forced image layers.

Reply:
I'll open a new thread for the default value.

Reply:
I changed default value of `forced_image_creation_limit` to 10. With current configuration parameters it limits growth of storage size caused by generation of image layers wanted by GC to 1280Mb/hour. With 7 day PiTR interval maximal storage growth is 7*24*1280=210Gb. Then GC should be able to remove some old images and storage size will be reduced. Please notice that this estimation (210Gb) is certainly not for all tenants - it is true for huge tenats with  Tb storage size. For small tenant this storage growth will e much smaller.